### PR TITLE
[sfputilbase.py] Don't try to print EEPROM sysfs file name if we failed to read from it

### PR DESCRIPTION
**Pull request by chiourung**
What I did:
    Remove undefined variable 'sysfs_sfp_i2c_client_eeprom_path'

Signed-off-by: chiourung_huang <chiourung_huang@edge-core.com>

### CODE REVIEW

File: sonic_platform_base/sonic_sfp/sfputilbase.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def _read_eeprom_specific_bytes(self, sysfsfile_eeprom, offset, num_bytes):
             sysfsfile_eeprom.seek(offset)
             raw = sysfsfile_eeprom.read(num_bytes)
         except IOError:
-            print("Error: reading sysfs file %s" % sysfs_sfp_i2c_client_eeprom_path)
+            print("Error: reading sysfs file %s")
```
Comment (jleveque) [inline sonic_platform_base/sonic_sfp/sfputilbase.py:324]: Why leave the `%s` formatter in the message with no variable to format?

Is the issue that `sysfs_sfp_i2c_client_eeprom_path` can be `None`? If so, I'd prefer you perform a check for `None` and add the path to the message if the path is not `None`.

Comment (chiourung) [reply sonic_platform_base/sonic_sfp/sfputilbase.py]: sysfs_sfp_i2c_client_eeprom_path is undefined.

Comment (jleveque) [reply sonic_platform_base/sonic_sfp/sfputilbase.py]: By "undefined" do you mean it is `None`? It should get defined in `_get_port_eeprom_path()`. Can you provide the backtrace from the crash you're seeing?

Comment (chiourung) [reply sonic_platform_base/sonic_sfp/sfputilbase.py]: Nov  3 17:36:14.436154 sonic INFO pmon#supervisord: xcvrd   File "/usr/local/lib/python2.7/dist-packages/sonic_sfp/sfputilbase.py", line 323, in _read_eeprom_specific_bytes
Nov  3 17:36:14.436168 sonic INFO pmon#supervisord: xcvrd     print("Error: reading sysfs file %s" % sysfs_sfp_i2c_client_eeprom_path)
Nov  3 17:36:14.436177 sonic INFO pmon#supervisord: xcvrd NameError: global name 'sysfs_sfp_i2c_client_eeprom_path' is not defined
Nov  3 17:36:15.445584 sonic INFO pmon#supervisor-proc-exit-listener: Process xcvrd exited unxepectedly. Terminating supervisor...

Comment (jleveque) [reply sonic_platform_base/sonic_sfp/sfputilbase.py]: I see. You've reached the exception case _because_ `sysfs_sfp_i2c_client_eeprom_path` is not defined, so printing it out doesn't make sense.

File: sonic_platform_base/sonic_sfp/sfputilbase.py [hunk 2]
```diff
@@ -321,7 +321,7 @@ def _read_eeprom_specific_bytes(self, sysfsfile_eeprom, offset, num_bytes):
             sysfsfile_eeprom.seek(offset)
             raw = sysfsfile_eeprom.read(num_bytes)
         except IOError:
-            print("Error: reading sysfs file %s" % sysfs_sfp_i2c_client_eeprom_path)
+            print("Error: reading sysfs file")
```
Comment (jleveque) [inline sonic_platform_base/sonic_sfp/sfputilbase.py:324]: Suggest to change message to `Error: reading EEPROM sysfs file`

**Pull request merged**